opm/simulators/linalg/setupPropertyTree.cpp

```diff
@@ -111,7 +115,7 @@ setupPropertyTree(FlowLinearSolverParameters p, // Note: copying the parameters
     // No valid configuration option found.
     OPM_THROW(std::invalid_argument,
               conf + " is not a valid setting for --linear-solver-configuration."
```

Comment:
I see this comment is outdated, the name of the option was changed to `--linear-solver` a long time ago. Can you drive-by-fix this too?

Reply:
this should be done in a separate PR and marked for the release.

Reply:
Good point. I will create such a fixup PR.